airbyte-integrations/connectors/source-qualaroo/bin/.dockerignore

```diff
@@ -0,0 +1,7 @@
+*
+!Dockerfile
+!Dockerfile.test
+!main.py
+!source_qualaroo
+!setup.py
```

Comment:
are these files duplicated ?

Reply:
yeah used another connector as boilerplate, forgot to remove